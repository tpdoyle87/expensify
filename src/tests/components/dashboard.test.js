import Dashboard from '../../components/dashboard'
import React from 'react'
import { shallow } from 'enzyme'

test('it should render the dashboard page', () => {
  const wrapper = shallow(<Dashboard />)
  expect(wrapper).toMatchSnapshot();
})
