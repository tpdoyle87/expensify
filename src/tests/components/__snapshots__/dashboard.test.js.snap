// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the dashboard page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
